index.bs

```diff
@@ -1850,6 +1942,103 @@ stream lacks this WebVTT file signature, then the parser aborts.</p>
  <li><p>If <var>line</var> is not the empty string, then jump back to the step labeled
  <i>header</i>.</p></li>
 
+ <li><p><i>Style block loop</i>: <a spec=html>Collect a sequence of characters</a> that are U+000A
```

Comment:
> On Oct 22, 2015, at 9:42 , Philip Jägenstedt <notifications@github.com> wrote:
> 
> In index.bs:
> 
> > @@ -1850,6 +1942,103 @@ stream lacks this WebVTT file signature, then the parser aborts.</p>
> >   <li><p>If <var>line</var> is not the empty string, then jump back to the step labeled
> >   <i>header</i>.</p></li>
> >  
> > + <li><p><i>Style block loop</i>: <a spec=html>Collect a sequence of characters</a> that are U+000A
> 
> Rather than a style block loop followed by a cue loop, I think we should have a single loop that handles style blocks and cues interleaves, but discards style blocks that come after any cue. This will make it trivially easy to allow interleaved blocks when we want them, and gives us high confidence that we won't stumble upon some mistake in the structure of the parser only then, when it may be too late to change.
> 

sounds good and prudent


David Singer
Manager, Software Standards, Apple Inc.